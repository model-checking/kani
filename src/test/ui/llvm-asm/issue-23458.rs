#![feature(llvm_asm)]
#![allow(deprecated)] // llvm_asm!
// compile-flags: -Ccodegen-units=1
// build-fail
// only-x86_64

fn main() {
    unsafe {
        llvm_asm!("int $3"); //~ ERROR too few operands for instruction
                             //~| ERROR invalid operand in inline asm
    }
}
