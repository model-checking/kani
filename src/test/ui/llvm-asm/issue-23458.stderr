error: invalid operand in inline asm: 'int $3'
  --> $DIR/issue-23458.rs:9:9
   |
LL |         llvm_asm!("int $3");
   |         ^

error: too few operands for instruction
  --> $DIR/issue-23458.rs:9:9
   |
LL |         llvm_asm!("int $3");
   |         ^
   |
note: instantiated into assembly here
  --> <inline asm>:1:2
   |
LL |     int 
   |     ^

error: aborting due to 2 previous errors

